includes/wikia/nirvana/WikiaView.class.php

```diff
@@ -295,7 +316,6 @@ protected function renderHtml() {
 				wfProfileOut(__METHOD__);
 
 				return $result;
-				break;
 			case WikiaResponse::TEMPLATE_ENGINE_PHP:
 			default:
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce this "switch/case" number of lines from 20 to at most 10, for example by extracting code into function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1151)